minutes/2025-05-02-psc-meeting.md

```diff
@@ -0,0 +1,71 @@
+---
+minutes-recorded-by: "@gadomski"
+---
+
+# 2025-05-02 STAC PSC meeting
+
+In-person in Snowbird, UT after the [2025 CNG Conference](https://conference.cloudnativegeo.org/CNGConference2025).
+In attendance:
+
+- @PowerChell
+- @matthewhanson
+- @m-mohr
+- @jsignell
+- @gadomski
+- @alexgleith
+
+Not in attendance:
+
+- @emmanuelmathot
+
+## Agenda
+
+Agenda was decided during the first part of the meeting via suggestions then triage, in the interest of time.
+
+- Welcome Alex!
+- PSC purpose
+- Communications
+- Community meetings
+- STAC as a OGC community standard
+- Sprint(s)
+
+## Minutes
+
+### Funding and PSC purpose
+
+- Discussed STAC PSC's relationship with the [Cloud-Native Geospatial (CNG) Forum](https://cloudnativegeo.org/), and whether CNG should work like [NumFocus](https://numfocus.org/) and have "sponsored projects" (e.g. [numpy](https://numfocus.org/project/numpy)).
+  (note: NumFocus provides funding and resources as well)
+- Decided on five overarching goals for the PSC:
+  - Interface with CNG, Radiant Earth, and any other organizations and standards bodies (e.g. [OGC](https://www.ogc.org/))
+  - Create and publish a roadmap for the STAC spec, the STAC API spec, and the STAC ecosystem
+  - Filter and publicly identify issues and problems in the STAC ecosystem, and moderate resolutions to those problems when necessary
+  - Identify and solve core specification issues
+  - Provide and participate in forums for the community to discuss and work with STAC
+
+### Communications
+
+- Accepted motion (@gadomski, second @matthewhanson) move "permanent, threaded discussions" to the [CNG Discourse](https://cng.discourse.group) and disable Github discussions for [stac-spec](https://github.com/radiantearth/stac-spec/discussions) and [stac-api-spec](https://github.com/radiantearth/stac-api-spec/discussions)
+  - Other organizations and repositories can choose to follow suit, but they don't have to
+  - @gadomski will coordinate the transition
+- @PowerChell will ask CNG to open Slack so that we can move STAC discussions there (currently on [Gitter](https://app.gitter.im/#/room/#SpatioTemporal-Asset-Catalog_Lobby:gitter.im))
+- @jsignell will update the website to point to the community meetings and the appropriate CNG Discourse channel
+- @PowerChell to schedule next PSC meeting, thereafter next meeting will be scheduled before ending current meeting
+- Accepted motion (@matthewhanson, second @jsignell) to delete the STAC Twitter account
+  - @PowerChell to coordinate transfer ownership of STAC Twitter, STAC LinkedIn from CNG
+  - @PowerChell to delete STAC Twitter
+  - @PowerChell to coordinate transfer of STAC domain to STAC PSC
+  - @m-mohr to set up STAC extension domain redirect
```

Comment:
What's the intended change with the extension domain?

Reply:
To replace stac-extensions.github.io slowly with extensions.stacspec.org to get more independant of github, but of course we'll keep stac-extensions.github.io as redirect as long as github exists.